typescript/test/testdata/translate/test/gapic-translation_service-v3beta1.ts.baseline

```diff
@@ -469,8 +484,9 @@ describe('v3beta1.TranslationServiceClient', () => {
             });
             // Mock request
             const request: protosTypes.google.cloud.translation.v3beta1.IListGlossariesRequest = {};
+            request.parent = '';
             // Mock response
```

Comment:
Can I get some context of adding `request.parent` and `expectedResponse = {response: 'data'}`? It seems not related to header fix.

Reply:
It partly is - the header fix triggered adding `request.parent` which made `request` not equal to `expectedResponse` and the test started failing. The fail is expected: the stream is supposed to emit a response, not a request. So I fixed the test.

Reply:
I see, thanks for clarification!